Src/DependencyCollector/Shared/Implementation/ProfilerHttpProcessing.cs

```diff
@@ -472,6 +472,22 @@ private void OnEnd(object exception, object thisObj, object returnValue)
                                         telemetry.Target += " | " + targetAppId;
                                     }
                                 }
+
+                                string targetRoleName = null;
+                                try
+                                {
+                                    targetRoleName = responseObj.Headers.GetNameValueHeaderValue(RequestResponseHeaders.RequestContextHeader, RequestResponseHeaders.RequestContextTargetRoleNameKey);
+                                }
+                                catch (Exception ex)
+                                {
+                                    AppMapCorrelationEventSource.Log.GetComponentRoleNameHeaderFailed(ex.ToInvariantString());
+                                }
+
+                                if (!string.IsNullOrEmpty(targetRoleName))
+                                {
+                                    telemetry.Type = RemoteDependencyConstants.AI;
+                                    telemetry.Target += " | roleName:" + targetRoleName;
```

Comment:
do we need a space after `:`? you have spaces before and after `|` so why not here?